climada/util/config.py

```diff
@@ -48,14 +47,6 @@ def remove_handlers(logger):
 LOGGER.addHandler(CONSOLE)
 
 
-def setup_logging(log_level='DEBUG'):
-    """Setup logging configuration"""
-    remove_handlers(LOGGER)
-    LOGGER.propagate = False
-    LOGGER.setLevel(getattr(logging, log_level))
-    LOGGER.addHandler(CONSOLE)
```

Comment:
Apart from updating the log-level (which I now moved to the end of the file), this only repeats configuration steps that are already applied in the lines above.